ledger/stores/chunked_file_store.py

```diff
@@ -23,16 +25,27 @@ class ChunkedFileStore(FileStore):
 
     firstChunkIndex = 1
 
-    @staticmethod
-    def _fileNameToChunkIndex(fileName):
+    def _fileNameToChunkIndex(self, fileName) -> int:
```

Comment:
Please add tests (unit tests) for this method.
Test contract (valid inputs, expected behaviour, such a structure of dirs and files, number of files in a dir, number of zero padding).
Test negative cases.